### ISSUE TITLE
remove codeception

### ISSUE THREAD

**Issue by ruskiyos** [finicprint/diaclone#40]
Looking at our tests, they are ALL unit tests.
There is really no point in loading in all of `codeception` module if we are using only simple assertions.
`phpunit` weighs so much less and is not much different than `codeception` (since codeception just wraps phpunit)

**Comment by alexcoffee** [comment]
You're right and I'm ok with removing codeception. In reality though we don't have that many tests yet, do the gains justify the cost of converting?

**Comment by iampersistent** [comment]
Why not `codeception`? I don't quite understand what weight has to do with the decision.

**Comment by alexcoffee** [comment]
For us it doesn't make sense because we don't care about weight and we are used to codeception. But if we wanted to make it into a public library then pure `phpunit` would be the more common dependency.

**Comment by ruskiyos** [comment]
if I have a project that uses only `phpunit`, why should `codeception` be a required (dev) package.

**Comment by iampersistent** [comment]
@alexcoffee I don't think codeception is a deterrent to people using a library. 

@ruskiyos This project *is* using codeception. :)

**Comment by iampersistent** [comment]
I don't care enough to die on this hill, but I don't see any value in changing it.

**Comment by alexcoffee** [comment]
lol I don't care enough to die on the hill either and I don't see the benefit in changing it for us. But I do see the value in "don't require more than you use".

**Comment by ruskiyos** [comment]
this is a public packagist repo, not a project lol. that's why I want to remove the dependency on codeception. if someone has codeception installed, they will automatically have phpunit, but not the other way around
When I released our autopilot plugin to the world, there were some people who contributed but were not familiar with codeception. i think
